src/quo2/components/tabs/account_selector.cljs

```diff
@@ -31,9 +31,12 @@
   {:height 16
    :width  16})
 
-(def account-emoji-container
-  {:background-color (colors/custom-color :purple 50)
-   :padding          8
+(defn account-emoji-container
+  [theme]
+  {:background-color (colors/theme-colors (colors/custom-color :purple 50)
```

Comment:
oh this was an oversight in pr review - should be using `customization-color` prop here and not `:purple`